buildSrc/subprojects/ide/src/main/kotlin/org/gradle/gradlebuild/ide/IdePlugin.kt

```diff
@@ -77,230 +56,120 @@ limitations under the License."""
+            val libsRepo = rootProject.layout.buildDirectory.dir("repo").get().asFile.absolutePath
+
+            val vmParameter = mutableListOf(
+                "-ea",
+                "-Dorg.gradle.docs.releasenotes.rendered=$releaseNotesFile",
+                "-DintegTest.gradleHomeDir=\$MODULE_DIR\$/build/integ test",
+                "-DintegTest.gradleUserHomeDir=$gradleUserHomeDir",
+                "-DintegTest.gradleGeneratedApiJarCacheDir=\$MODULE_DIR\$/build/generatedApiJars",
+                "-DintegTest.libsRepo=$libsRepo",
+                "-Dorg.gradle.integtest.daemon.registry=$daemoRegistryDir",
+                "-DintegTest.distsDir=$distsDir",
+                "-Dorg.gradle.public.api.includes=${PublicApi.includes.joinToString(":")}",
+                "-Dorg.gradle.public.api.excludes=${PublicApi.excludes.joinToString(":")}",
+                "-Dorg.gradle.integtest.executer=embedded",
+                "-Dorg.gradle.integtest.versions=default",
+                "-Dorg.gradle.integtest.testkit.compatibility=current",
+                "-Xmx512m",
+                "--add-opens", "java.base/java.lang=ALL-UNNAMED",
+                "--add-opens", "java.base/java.lang.invoke=ALL-UNNAMED",
+                "--add-opens", "java.base/java.lang.reflect=ALL-UNNAMED",
+                "--add-opens", "java.base/java.util=ALL-UNNAMED",
+                "--add-opens", "java.prefs/java.util.prefs=ALL-UNNAMED"
+            )
+            vmParameters = vmParameter.joinToString(" ") {
+                if (it.contains(" ") || it.contains("\$")) "\"$it\""
+                else it
             }
         }
     }
 
     private
-    fun ProjectSettings.configureCopyright() {
-        copyright {
-            useDefault = GradleCopyright.profileName
-            profiles {
-                create(GradleCopyright.profileName) {
-                    notice = GradleCopyright.notice
-                    keyword = GradleCopyright.keyword
-                }
-            }
-        }
-    }
-
-    private
-    fun ProjectSettings.configureCompilerSettings(project: Project) {
-        compiler {
-            processHeapSize = javaCompilerHeapSpace
-            useReleaseOption = false
-        }
-        groovyCompiler {
-            excludes {
-                file("${project.rootProject.projectDir.absolutePath}/subprojects/plugins/src/test/groovy/org/gradle/api/internal/tasks/testing/junit/JUnitTestClassProcessorTest.groovy")
-            }
-        }
-    }
+    fun Project.docsProject() =
+        project(":docs")
 
     private
-    fun ProjectSettings.configureInspections() {
-        inspections {
-            disabledInspections.forEach { name ->
-                create(name) {
-                    enabled = false
-                }
-            }
-        }
-    }
-
+    val lang: String
+        get() = System.getenv("LANG") ?: "en_US.UTF-8"
```

Comment:
:x: should use `providers.environmentVariable("LANG")` in order to let instant execution consider that environment variable as a build logic input